src/scene/scene_phongmaterial.js

```diff
@@ -526,6 +535,7 @@ pc.extend(pc, function () {
             this._mapXForms = [];
 
             var useTexCubeLod = device.useTexCubeLod;
+            var useDp = !device.extTextureLod; // no basic extension? likely slow device, force dp

```

Comment:
No, lack of extension means likely slow device (except normal device running IE).